dart/test/event_processor/exception/io_exception_event_processor_test.dart

```diff
@@ -69,6 +69,7 @@ void main() {
       );
       expect(event.exceptions?.first.mechanism?.type, 'OSError');
       expect(event.exceptions?.first.mechanism?.meta['errno']['number'], 54);
+      expect(event.exceptions?.first.mechanism?.handled, isNull);
```

Comment:
You can undo this change in favor of https://github.com/getsentry/sentry-dart/pull/1216